Source/Plugin.BLE.Android/Device.cs

```diff
@@ -142,8 +140,7 @@ public void Disconnect()
                 IsOperationRequested = true;
 
                 _gatt.Disconnect();
-
-                CloseGatt();
```

Comment:
and we never want to close gatt? Maybe after disconnect finally happens?

Reply:
That is a good point. I have changed the implementation to do that. Would you like to take an extra look at the new stuff?

Reply:
looks good (if it works?)

Reply:
Tested it on the Hauweii I have and it works on that.